app/assets/javascripts/admin/addon/controllers/admin-badges/show.js

```diff
@@ -80,6 +79,17 @@ export default class AdminBadgesShowController extends Controller {
     return this.model.system;
   }
 
+  @action
+  postHeaderDescription(data = {}) {
```

Comment:
are those two actions ever called with a null-ish arg?

Reply:
Yes, it was a surprise to me too. I need to look how that can happen.

Reply:
I have been checking this enough... and couldn't repro the empty data anymore, it was probably related to another error. I pushed a change to remove the `= {}`